test/graph/graph-mary-manipulation-test.js

```diff
@@ -39,62 +46,60 @@ describe('Graph manipulation for `mary` stream', function() {
         });
         it('node without an id will get a generated id', function() {
             node.setType('TOK');
-            graph.addNode(node);
             // IDs should increment consecutively.
             assert.equal(node.getId(), (parseInt(graph._generateNodeId()) - 1).toString());
         });
         it('node will be added to the graph', function() {
             var preAddTotalCount = graph.getNodes().length;
             var preAddTypeCount = graph.getNodesByType('TOK').length;
-            node.setType('TOK');
+            node = new Node('900000', graph, 'TOK');
             graph.addNode(node);
             assert.equal(node, graph.getNodeById(node.getId()));
             assert.equal(preAddTotalCount, graph.getNodes().length - 1);
             assert.equal(preAddTypeCount, graph.getNodesByType(node.getType()).length - 1);
         });
         describe('sequence trait', function() {
+            beforeEach(createGraph);
```

Comment:
You are correct, moved the describe block out to avoid the double run. It's not quite as semantically clean but I think the benefit of clean graphs on every it test outweighs having the nicer test structure.